pages/account/customer/ovhcloud-account-connect-saml-okta/guide.en-gb.md

```diff
@@ -0,0 +1,169 @@
+
+And for Group Attribute Statements
+- **Name**: `groups` and **Filter**: `Matches regex:.*` (Adapt the filter if you want to be more specific)
+
+Click `Next`{.action}.
+
+![Add SAML application, step 5](images/OKTA_add_application_step5.png){.thumbnail}
+
+In the "Feedback" step, select the according option and click `Finish`{.action}.
+
+Then open the application and go to the "Assignments" tab and assign users or groups to the application.
+
+![Assign users](images/OKTA_add_user.png){.thumbnail}
+
+Before going to the next section, go to the "Sign On" tab, and access to the **Metadata URL** and save the XML provided
+
+![Retrieve metadata](images/OKTA_retrieve_metadata.png){.thumbnail}
+
+Your OKTA service now trusts OVHcloud as a service provider. The next step is to ensure that the OVHcloud account trusts your OKTA as an identity provider.
+
+### Registering OKTA into the OVHcloud account and configuring the connection
+
+To add OKTA as a trusted identity provider, you need to provide the identity provider metadata in the [OVHcloud Control Panel](https://www.ovh.com/auth/?action=gotomanager&from=https://www.ovh.co.uk/&ovhSubsidiary=GB).
+
+Once logged in, click your profile at the top right.
+
+![Top menu OVHcloud](images/ovhcloud_top_menu.png){.thumbnail}
+
+Click on your name to access your profile management page.
+
+![OVHcloud User Information](images/ovhcloud_user_infos.png){.thumbnail}
+
+Open the `User Management`{.action} tab.
+
+![OVHcloud menu profile](images/ovhcloud_profile_menu.png){.thumbnail}
+
+Click the `SSO connection`{.action} button.
+
+![OVHcloud SSO connection step 1](images/ovhcloud_user_management_connect_sso_1.png){.thumbnail}
+
+Fill in the XML metadata of your OKTA service. Enter `groups` as the "Group Attribute Name". Click `Confirm`{.action}.
+
+![OVHcloud SSO connection step 2](images/ovhcloud_add_federation.png){.thumbnail}
+
+Now you need to retrieve your OKTA as identity provider, as well as default groups.
+
+![OVHcloud SSO connection step 3](images/ovhcloud_add_federation_success.png){.thumbnail}
+
+For more information, click on the link under “SSO Service URL”.
+
+![OVHcloud SSO connection step 4](images/ovhcloud_idp_details.png){.thumbnail}
+
+The `...`{.action} button allows you to update or delete the SSO, and view its details.
+
+![OVHcloud SSO connection step 5](images/ovhcloud_user_management_connect_sso_5.png){.thumbnail}
+
+Your OKTA is now considered a trusted identity provider. However, you still need to add groups to your OVHcloud account.
+
+> [!warning]
+> If you try to connect via SSO at this point, you will probably receive a `Not in valid groups` error message.
+>
+> That is because your OVHcloud account checks whether the authenticating user belongs to an existing group on the account.
+>
+
+You must then assign **roles** to OKTA user groups at OVHcloud. Otherwise, your OVHcloud account does not know what the user is allowed to do and, by default, no rights are assigned.
```

Comment:
```suggestion
You must then assign **roles** to Okta user groups at OVHcloud. Otherwise, your OVHcloud account does not know what the user is allowed to do and, by default, no rights are assigned.
```